x-pack/plugin/esql/build.gradle

```diff
@@ -65,6 +65,7 @@ tasks.named("test").configure {
           String tag = it[2]
           boolean isJson = it[3]
           String allExamples = new File("${projectDir}/qa/testFixtures/src/main/resources/${file}").text
+            .replaceAll(System.lineSeparator(), "\n")
```

Comment:
FYI: As we're searching/replacing/indexing-by '\n' at multiple parts here, decided to just replace any non-\n separator to \n here at the start

Reply:
Fine by me.